packages/cx-editors/src/components/RichText/tokens/cxRichText.ts

```diff
@@ -109,26 +111,43 @@ const Default = asCxTokenSpec()({
   },
 });
 
-const Copyright = asCxTokenSpec()({
+const Basic = asCxTokenSpec()({
   ...Default,
+  Core: pick(Default.Core, 'paragraph', 'Bold', 'Underline', 'Link', 'SuperScript'),
+  Components: pick(Default.Components, 'Bold', 'Underline', 'SuperScript'),
+});
+
+const Copyright = asCxTokenSpec()({
+  ...Basic,
   Spacing: {
     paragraph: 'mx-9 py-9 md:mx-0 md:mb-4 md:p-0 lg:mt-2 lg:mb-0 lg:py-0',
   },
   Theme: {
     paragraph: as(
       cxColor.TextPrimaryFooterCopy,
       cxFontSize.XS,
-      'border-white-400 border-t border-b md:border-0 lg:text-m-xs',
+      cxTextDecoration.Normal,
+      'border-white-400 border-t border-b md:border-0',
```

Comment:
The border should go on copyright slot -- we don't want a border between every paragraph.

Reply:
Border changed to the copyright slot